clickhouse.go

```diff
@@ -6,15 +6,15 @@ import (
 )
 
 func init() {
-	sql.Register("chhttp", new(chDriver))
+	sql.Register("chhttp", new(ChDriver))
 }
 
-// chDriver implements sql.Driver interface
-type chDriver struct {
+// ChDriver implements sql.Driver interface
+type ChDriver struct {
```

Comment:
Can we name it `Driver` so during usage we specify `clickhouse.Driver` and not `clickhouse.ChDriver`?